en/starter/generator/command_line_tools.md

```diff
@@ -45,4 +43,4 @@ Allows you to define a data model for a CRUD ([Create-Read-Update-Delete](core_c
 
 > ![tip] Note that you must run `kitura create` before creating your first model.
 
-
+> ![info] Some additional, advanced features of the generator can be accessed by installing the [Advanced Command-line interface](command_line_tools_for_linux).
```

Comment:
I'm not sure what we should call this but I don't like the phrase 'Advanced Command-line interface'

Reply:
We could call it what it is, 'Swift Server Generator', instead.

Reply:
Agreed, call it what it is.